src/main/java/seedu/address/logic/commands/SetRsvpCommand.java

```diff
@@ -0,0 +1,83 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.RsvpStatus;
+
+/**
+ * Set RSVP status for guests
+ */
+public class SetRsvpCommand extends Command {
+    public static final String COMMAND_WORD = "SetRsvp";
```

Comment:
Should the command word be in lowercase to standardise with the rest of the commands?